test/Dockerfile

```diff
@@ -0,0 +1,5 @@
+from golang:1.14
```

Comment:
maybe can use the same go version that is being used in the go mod: https://github.com/mattermost/awat/blob/main/go.mod#L3

also I would use the docker keywords in all caps, like  `FROM`  / `WORKDIR`...

Reply:
I always hated the all-caps keywords in Dockerfiles, personally; reminds me of BASIC. I'll change it anyway.